tests/components/webhook/test_init.py

```diff
@@ -143,6 +145,37 @@ async def handle(*args):
     assert hooks[0][2].method == "HEAD"
 
 
+async def test_webhook_local_only(hass, mock_client):
+    """Test posting a webhook with local only."""
+    hooks = []
+    webhook_id = hass.components.webhook.async_generate_id()
```

Comment:
Should this not be done with a normal import? That is at least the feedback I got on my pull request :)

Reply:
ha yes. I copied it from another test but will fix.